plutus-ledger-api/src/Plutus/V1/Ledger/Scripts.hs

```diff
@@ -325,9 +328,8 @@ mintingPolicyHash vl = MintingPolicyHash $ BA.convert h' where
 
 -- | Information about the state of the blockchain and about the transaction
 --   that is currently being validated, represented as a value in 'Data'.
-newtype Context = Context Data
-    deriving stock (Generic, Haskell.Show)
-    deriving anyclass (ToJSON, FromJSON)
+newtype Context = Context BuiltinData
+    deriving (ToJSON, FromJSON) via PLC.Data
```

Comment:
Could you derive Pretty this way too?

Reply:
Yes, do you think it's useful? Doesn't hurt, I guess.